docs/user_guide/kernel_methods.rst

```diff
@@ -6,26 +6,301 @@
 
-Training of quantum kernels:
-----------------------------
+enriched with the *Hilbert-Schmidt inner product* as the feature space of a quantum model and 
+state [1]. Thus, in quantum computing, access to the Hilbert space of quantum states is given by 
+measurements. In contrast to conventional kernel methods, the central task for quantum kernel 
+methods is to find sufficiently well suited quantum feature maps for a given data set, while in 
+classical kernel methods one never only treats feature maps implicitly by selecting a proper
+kernel function for a given problem.
 
+
+.. currentmodule:: squlearn.kernel.ml
+
+High-Level methods that employ quantum kernels
+----------------------------------------------
+In general, kernel methods refer to a collection of pattern analysis algorithms that use kernel 
+functions to operate in a high-dimensional feature space. Probably, the most famous representative
+of these kernel-based algorithms is *Support Vector Machines (SVMs)*. Kernel methods are most 
+commonly used in a supervised learning framework for either classification or regression tasks. 
+But, there are use-cases for kernel-based unsupervised algorithms too; however, these are (currently)
+not covered by sQUlearn. 
+
+In the NISQ era (no access to fast linear algebra algorithms such as HHL), the basic notion of 
+quantum kernel methods is to merely compute the kernel matrix with a quantum computer and 
+subsequently pass it to an conventional kernel algorithms. For this task, sQUlearn provides a 
+convenient workflow by either wrapping the corresponding scikit-learn estimators or by 
+independetly implementing them analougsly, adapted to the needs of quantum kernel methods. 
+sQUlearn offers SVMs for both classifciation (QSVC) and regression (QSVR) tasks, Gaussian 
+Processes (GPs) for both classification (QGPC) and regression (QGPR) tasks as well as a 
+quantum kernel ridge regression routine (QKRR).
+
+Classification
+##############
+
+In terms of classification tasks, sQUlearn provides:
+
+.. autosummary::
+   :nosignatures:
+
+   QSVC
+   QGPC
+
+We refer to the documentations and examples of the respective methods for in-depth information
+and user guidelines.
+
+Regression
+##########
+
+In terms of regression tasks, sQUlearn provides:
+
+.. autosummary::
+   :nosignatures:
+
+   QSVR
+   QKRR
+   QGPR
+
+We refer to the documentations and examples of the respective methods for in-depth information
+and user guidelines.
+
+
+Methods to evaluate quantum kernels
+-----------------------------------
+sQUlearn provides two methods to evaluate quantum kernels: Fidelity Quantum kernels (FQKs) and 
+Projected Quantum kernels (PQKs), which also represent the standard approaches to quantum kernel 
+methods in the literature.
+
+Central to both approaches is the embedding of data into the quantum Hilbert space by using
+quantum feature maps, which are nothing but parameterized quantum circuits. At this, the 
```

Comment:
Maybe simpler:

...by using quantum feature maps which are nothing but encoding quantum circuits. These can optionally be parametrized for optimally adjusting the quantum kernel...